### PR TITLE
Create virtual-box cheatsheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**

Virtual box cheatsheet continuing #2214

---

Instant Answer Page: https://duck.co/ia/view/virtual_box_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Virtual Box Cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/virtual_box_cheat_sheet_mobile.png?4v8jrNVxxG)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Virtual Box Cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/virtual_box_cheat_sheet_mobile.png?iFNxHktJYK)

**Comment by zekiel** [comment]
@gautamkrishnar thanks for this! 2 things come to mind that could improve this: 

1. Can you update the section titles to reflect what the shortcuts in each section do? For example:   `Commands `
`Navigation`

2. Can you provide a source for this info?

File: share/goodie/cheat_sheets/json/virtual-box.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "id": "virtual_box_cheat_sheet",
+  "name": "Oracle Virtual Box",
+  "description": "Keyboard shortcuts for virtualbox",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/virtual-box.json]: Probably want to be consistent and use `VirtualBox` where appropriate.

**Comment by gautamkrishnar** [comment]
Sure...

**Comment by gautamkrishnar** [comment]
@zekiel do you have any suggestions on sections? I can't find a relevant source too...

**Comment by zekiel** [comment]
@gautamkrishnar sections seem ok now

on the source -- I can't find any with a quick search. Are you getting these from the UI itself? It's ok to merge but it is a bit odd to not have a source defined. We'll need a better way to handle that long-term as well.

**Comment by gautamkrishnar** [comment]
@zekiel  I can't find a reliable source for this ia... Maybe let the other community members do...
Anyone like to improve this IA please create a new PR @duckduckgo/duckduckhack-contributors

**Pull request merged**